src/components/institution/SelectInstitutionForm.tsx

```diff
@@ -132,8 +137,12 @@ export const SelectInstitutionForm = ({
                     }
                   }}
                   onChange={(_, value) => {
-                    resetForm();
+                    resetForm({
+                      values: initialValuesOrganizationForm,
+                    });
```

Comment:
This is to ensure that when we cross out the value of the top field, the bottom field is also blanked out